core_lib/src/tool/penciltool.cpp

```diff
@@ -51,10 +53,7 @@ void PencilTool::loadSettings()
     properties.pressure = settings.value("pencilPressure", true).toBool();
     properties.stabilizerLevel = settings.value("pencilLineStabilization", StabilizationLevel::STRONG).toInt();
     properties.useAA = DISABLED;
-    properties.useFeather = true;
```

Comment:
It does nothing afaik, which is why it's been removed. We currently have 3 ways to check for feather state.. either useFeather which only have a real use for eraser where it's used for the toggle, we also have the  mPropertyEnabled[FEATHER] and the properties.feather value to check.

This was an attempt to clean some stuff up, although initially it was because i used the useFeather to indicate whether to draw the feather circle on the stroke tool but I changed that recently to use the  mPropertyEnabled[FEATHER] state instead

I can revert these changes if you think that's better? Then we can do a proper cleanup of our properties class at some other point.